test/service/test_request_builder.py

```diff
@@ -78,9 +78,6 @@ def test_create_request(self):
                         }
                     ]
                 }
-            },
```

Comment:
Can you talk a little about why these can be removed?

Reply:
Yes, since we do not actually use the `query` route of the search request for this particular cases, it is not being specified in the request. It seem to have been a bug, you can read further [here](https://github.com/elastic/elasticsearch-dsl-py/issues/924).